src/compressor.h

```diff
@@ -0,0 +1,89 @@
+/*
+   ZynAddSubFX - a software synthesizer
+
+   Compressor.h - simple audio compressor macros
+   Copyright (C) 2016 Hans Petter Selasky
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+*/
+
+#pragma once
+
+#define	floatIsValid(x) ({			\
```

Comment:
`({ ... })` is a GNU C extension, not standard C++, so it won't work with all compilers. It'd be better to make this a (static inline) function.

Reply:
Fixed.